src/System.Private.Xml.Linq/tests/TreeManipulation/SaveWithFileName.cs

```diff
@@ -0,0 +1,287 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using CoreXml.Test.XLinq;
+using Microsoft.Test.ModuleCore;
+using XmlCoreTest.Common;
+using Xunit;
+
+namespace XLinqTests
+{
+    public class SaveWithFileName
+    {
+        private string _fileName = "SaveBaseline";
+
+        [Fact]
+        public void XDocumentSaveToFile()
+        {
+            SerializeWithSaveOptions(SerializeXDocumentSaveToFile, false, false, true);
```

Comment:
>SerializeWithSaveOptions(SerializeXDocumentSaveToFile, false, false, true); [](start = 12, length = 75)

style suggests writing like this whenever there's opaque parameters

 SerializeWithSaveOptions(SerializeXDocumentSaveToFile, testXText: false, testXElement: false, testXDocument: false)